src/src/PluginStructs/P073_data_struct.cpp

```diff
@@ -134,6 +134,20 @@ void P073_font_selector(const __FlashStringHelper *id, int16_t value) {
 
 # endif // ifdef P073_EXTRA_FONTS
 
+/**
+ * This function reverts the 7 databits/segmentbits so TM1637 and 74HC595 displays work with fonts designed for MAX7219.
+ * Dot/colon bit is still bit 8
+ */
+uint8_t P073_revert7bits(uint8_t character) {
+  uint8_t newCharacter = character & 0x80; // Keep dot-bit if passed in
```

Comment:
When you need to do this fast, you can use something like this:
```c++
uint8_t b = byte_to_reverse;
b = ((b * 0x0802LU & 0x22110LU) | (b * 0x8020LU & 0x88440LU)) * 0x10101LU >> 16;
```
See: https://graphics.stanford.edu/~seander/bithacks.html#BitReverseObvious